examples/file/stream/Stream.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using NationalInstruments.SystemLink.Clients.File;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.File
+{
+    /// <summary>
+    /// Example for the SystemLink File API that uploads a file to the
+    /// SystemLink server using an in-memory stream.
+    /// </summary>
+    class Stream
+    {
+        static void Main(string[] args)
+        {
+            /*
+             * See the configuration example for how a typical application
+             * might obtain a configuration.
+             */
+            var configuration = ExampleConfiguration.Obtain(args);
```

Comment:
We should either test that this works with cloud (and update the README accordingly, or pass `allowCloud: false` here.